test/widgets/drives_page_test.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/drives/pages/create_drive_page.dart';
+import 'package:motis_mitfahr_app/drives/pages/drives_page.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:supabase_flutter/supabase_flutter.dart';
+
+import '../util/factories/profile_factory.dart';
+import '../util/mock_server.dart';
+import '../util/pump_material.dart';
+import '../util/request_processor.dart';
+import '../util/request_processor.mocks.dart';
+
+void main() {
+  late Profile profile;
+  final MockRequestProcessor processor = MockRequestProcessor();
+
+  setUpAll(() async {
+    MockServer.setProcessor(processor);
+  });
+  setUp(() async {
+    profile = ProfileFactory().generateFake();
+    SupabaseManager.setCurrentProfile(profile);
```

Comment:
This is unnecessary right? You are not using the profile

Reply:
No i Need it, since i'm getting all rides/drives for current user. I could replace the id however With the supabase.currentProfile.id of you want

Reply:
Where? This whole test file does not use the profile. For the TripPageBuilder, I would agree.

Reply:
yes but when the cards get build the current Profile is accessed and it's easier if i know the id

Reply:
i gave the profile a purpose.